setup/pi/configure-web.sh

```diff
@@ -26,7 +26,17 @@ ln -s /mutable/archiveloop.log /var/www/html/
 ln -s /tmp/diagnostics.txt /var/www/html/
 mkdir /var/www/html/TeslaCam
 cp -rf "$SOURCE_DIR/teslausb-www/teslausb.nginx" /etc/nginx/sites-available
-ln -sf /etc/nginx/sites-available/teslausb.nginx /etc/nginx/sites-enabled/default
+cp -rf "$SOURCE_DIR/teslausb-www/teslausbsecure.nginx" /etc/nginx/sites-available
+
+# Setup nginx auth if WEB_USERNAME and WEB_PASSWORD have been defined
+if [ -z "${WEB_USERNAME}" -o -z "${WEB_PASSWORD}" ]
```

Comment:
I think this fails (and exits the script) with "unbound variable" if these are not set, because this script uses bash's `-u` option.

Reply:
Ahh 🤦 , sorry about that, I added a `WEB_AUTH_ENABLED` variable to prevent this issue.